config.default

```diff
@@ -326,6 +326,15 @@ movingAveragePoints = 12
 # regionIndicesToPlot should be an integer
 regionIndicesToPlot = 5
 
+# Data source to read for comparison.  There are three options
+# 1 - ERS SSTv3b -- Common in CVDP analysis -- 1854 - 2016
+# 2 - ERS SSTv4 -- Updated version of previous -- 1854 - 2016
+# 3 - HADIsst -- Hadley center analysis -- 1870 - 2016
+# Valid Choices -- Uncomment your choice 
+#observationData = /global/homes/l/lvroekel/HADIsst_nino34.nc
+observationData = /global/homes/l/lvroekel/ERS_SSTv3b_nino34.nc
+#observationData = /global/homes/l/lvroekel/ERS_SSTv4_nino34.nc
```

Comment:
My preference would be to ask Rich Neale whether we want to support 2 data sets (HADSST and ERsstv4) or just one, and if the latter then which data set in particular. Then, as @xylar says, let's just hardwire the name(s).
I don't think it is critical to have flags that let the user choose whether the obs are plotted or not, or whether one wants to plot the time series and/or the spectrum. I do not suppose plotting Nino34 diags for obs is going to take a big percentage of time. We do not support this kind of decision making for other diagnostics anyway.